packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -99,7 +102,8 @@ class CreateCommand extends FlutterCommand {
     );
     printStatus('Wrote $generatedCount files.');
 
-    printStatus('');
```

Comment:
Was intentional -- there's another linefeed right below the conditional below so there's a double paragraph break. That said, I realise I'm personally obsessed with vertical space. Will revert, but move it alongside the other one.